### ISSUE TITLE
Prevent auto-play/buffering works perfectly, however Prevent tab auto-play/buffering never work.

### ISSUE THREAD

**Issue by bugreport0** [YePpHa/YouTubeCenter#1534]

**Comment by blue-lightning** [comment]
It works for me, but I use middle mouse to open the video in a new tab. If you just click on a link and immediately switch to a new tab, the video will still be played like normal.

**Comment by bugreport0** [comment]
It's quite the opposite with me, middle-clicking a video always make it plays, even though "Prevent tab auto-play/buffering" is checked.

**Comment by blue-lightning** [comment]
After middle clicking, does the browser change focus (jump) to the new tab ? That's the default config in Chrome and Firefox, I think.

**Comment by bugreport0** [comment]
Nope, "middle-clicking" opens it as a background "unfocused " tab, and the video still plays.

**Comment by Eisys** [comment]
It works fine for me..
What browser are you using and what version of YTC are you using? That info always helps!
